### ISSUE TITLE
[Bug]Failed to convert maskrcnn_resnet50_fpn_v2 model from torchvision

### ISSUE THREAD

**Issue by dlod-openvino** [openvinotoolkit/openvino#13970]
##### System information (version)
- OpenVINO => 2022.2
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2017
- Problem classification: Model Conversion
- Framework: PyTorch
- Model name: maskrcnn_resnet50_fpn_v2

##### Detailed description
Failed to convert maskrcnn_resnet50_fpn_v2 model from torchvision

##### Steps to reproduce
**Step1** Using the following code to export maskrcnn_resnet50_fpn_v2 model from torchvision to ONNX format
```python
from torchvision.models.detection import maskrcnn_resnet50_fpn_v2, MaskRCNN_ResNet50_FPN_V2_Weights
import torch

weights = MaskRCNN_ResNet50_FPN_V2_Weights.DEFAULT
model = maskrcnn_resnet50_fpn_v2(weights=weights, progress=False)
model = model.eval()
dummy_input = torch.randn(1, 3, 800, 1333, device="cpu")
input_names = ["images"]
output_names = ['boxes', 'labels', 'scores', 'masks']
torch.onnx.export(model, 
                 dummy_input, 
                 "maskrcnn_resnet50_fpn_v2.onnx", 
                 verbose=True, 
                 input_names=input_names, 
                 output_names=output_names,
                 opset_version=11, 
                 )
```
After running the code, you could get the maskrcnn_resnet50_fpn_v2.onnx file
<img width="632" alt="1668328492395" src="https://user-images.githubusercontent.com/59691336/201513077-b452825e-6cfc-498c-b91d-30bccb12bdfd.png">

**Step2** Using the following code, you can do the inference successfully based on 
```python
import numpy as np
import onnxruntime as ort

ort_session = ort.InferenceSession("maskrcnn_resnet50_fpn_v2.onnx")

outputs = ort_session.run(
    None,
    {"images": np.random.randn(1, 3, 800, 1333).astype(np.float32)},
)
for out in outputs:
    print(out.shape)
```
<img width="543" alt="1668328662005" src="https://user-images.githubusercontent.com/59691336/201513188-a1513b0f-78be-4fd1-801f-2330abb8e9a8.png">

**Step3** Using the "mo -m maskrcnn_resnet50_fpn_v2.onnx", you will get the error while converting the model.
<img width="553" alt="1668328768996" src="https://user-images.githubusercontent.com/59691336/201513263-dd470815-2d03-4cec-a9b4-d5bfb51ea3e4.png">

>OpenVINO runtime version:       2022.2.0-7713-af16ea1d79a-releases/2022/2
Model Optimizer version:        2022.2.0-7713-af16ea1d79a-releases/2022/2
[ ERROR ]  -------------------------------------------------
[ ERROR ]  ----------------- INTERNAL ERROR ----------------
[ ERROR ]  Unexpected exception happened.
[ ERROR ]  Please contact Model Optimizer developers and forward the following information:
[ ERROR ]  While validating ONNX node '<Node(ConstantOfShape): ConstantOfShape_453>':
Check 'broadcast_shape_rank.compatible(1)' failed at C:\Jenkins\workspace\private-ci\ie\build-windows-vs2019\b\repos\openvino\src\core\shape_inference\include\broadcast_shape_inference.hpp:174:
While validating node 'v3::Broadcast Broadcast_5438 (Constant_5437[0]:i64{}, Constant_452[0]:i64{}) -> (dynamic...)' with friendly_name 'Broadcast_5438':
Broadcast shape rank must be 1, but has 0

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [*] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [*] There is reproducer code and related data files: images, videos, models, etc.

**Comment by andrei-kochin** [comment]
@dlod-openvino thanks for reaching OpenVINO!

@mlukasze looks like this something for your guys to check.
Thanks

**Comment by dlod-openvino** [comment]
Thank you @andrei-kochin @mlukasze , very much! Looking forward to the solution!:)

**Comment by mitruska** [comment]
Hello, @dlod-openvino thank you for reporting this issue!

Mentioned `maskrcnn_resnet50_fpn_v2.onnx` model has an empty tensor provided to the ONNX ConstantOfShape operator. 
According to the ONNX standard, it should lead to the scalar output shape.

I've created a PR to enable support for such case: 
- https://github.com/openvinotoolkit/openvino/pull/13994

**Comment by mitruska** [comment]
The PR https://github.com/openvinotoolkit/openvino/pull/13994 resolving this issue has been merged to the openvino master branch.
@dlod-openvino Let us know if there's anything more we can do to help!

**Comment by dlod-openvino** [comment]
> The PR #13994 resolving this issue has been merged to the openvino master branch. @dlod-openvino Let us know if there's anything more we can do to help!

Hi, @mitruska thank you very much for your replying! Do you mean OpenVINO 2022.3 will solve the problem of "ConstantOfShape with empty tensor input" ? Thanks!

**Comment by mitruska** [comment]
@dlod-openvino Indeed, it will be included in `2022.3` OpenVINO release.
